misc/ballerina-bindgen/src/main/java/org/ballerinalang/bindgen/command/BindgenCommand.java

```diff
@@ -38,6 +42,8 @@
 
     private static final PrintStream outStream = System.out;
     private static final PrintStream outError = System.err;
+    private Path sourceRootPath = Paths.get(System.getProperty("user.dir"));
```

Comment:
```suggestion
    private Path sourceRootPath = Paths.get(System.getProperty(USER_DIR));
```
Can we use `USER_DIR` from `BindgenConstants`?

Reply:
Yes, fixed.